src/server/run-server.ts

```diff
@@ -52,7 +52,7 @@ export function runServer(db: Knex, augur: Augur, controlEmitter: EventEmitter =
         } else {
           res.send({ status: ServerStatus.UP, universe });
         }
-      });
+      }).catch((e) => { throw e; });
```

Comment:
What does catch-just-to-rethrow do here? Wouldn't this be the normal behavior without a catch?

Reply:
yeah it's not a no-op to satisfy the new linter rules for dangling promises. should probably log instead of re-throwing.